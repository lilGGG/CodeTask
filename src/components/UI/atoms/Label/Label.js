import styled from 'styled-components';

const StyledLabel = styled.label`
    color: chocolate;
    padding: 0 0 4px 10px;
    line-height: 1.5;
    display: block;
`;

function Label({
  Text
}) {
  return (
     <> 
        <StyledLabel>
            {Text}
        </StyledLabel>
      </>
    );
}

export default Label;
