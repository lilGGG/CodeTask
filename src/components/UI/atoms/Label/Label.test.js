import React from "react";
import Label from "./Label";
import ReactDom from "react-dom";
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { create, act } from "react-test-renderer";
import { shallow, mount, render } from "enzyme";

describe('Label test', () => {

    it('renders snapshot', () => {
      const tree = renderer.create(<Label Text="title text"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });

    //   it("Should test component", () => {
    //     const wrapper = shallow(<Image SrcImage="https://static.tvmaze.com/uploads/images/medium_portrait/24/60941.jpg"/>);
    //     expect(wrapper).toMatchSnapshot();
    //   });
      
    // it('renders correctly', () => {
    //     const tree = renderer.create(<Image />).toJSON();
    //     expect(tree).toMatchInlineSnapshot();
    //   });
  });
  
