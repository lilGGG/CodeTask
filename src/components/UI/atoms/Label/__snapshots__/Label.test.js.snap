// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label test renders snapshot 1`] = `
<label
  className="sc-bczRLJ ZKYHI"
>
  title text
</label>
`;
